### ISSUE TITLE
no working

### ISSUE THREAD

**Issue by mswiatlo** [dguest/pandamonium#15]
```
~$ lsetup "panda 0.6.18" python
************************************************************************
Requested:  python panda ...
 Setting up python 2.7.4-x86_64-slc6-gcc48 ...
  Skipping: python 2.7 x86_64 already setup.
 Setting up panda 0.6.18 ...
>>>>>>>>>>>>>>>>>>>>>>>>> Information for user <<<<<<<<<<<<<<<<<<<<<<<<<
************************************************************************
~$ panda-resub-taskid 79174
Failed to load PandaClient, please set up locally
```

Am I being a dummy? what's wrong?

(I kinda expected the latest panda to not work because of the changes to pbook, but I"m surprised older ones die now too)

**Comment by dguest** [comment]
oh dear, this is the latest version of pandamonium?

**Comment by dguest** [comment]
also, it _should_ work with the latest version of panda too, since I merged in @alexarmstrongvi's changes.

**Comment by mswiatlo** [comment]
whoops yes i'm a dummy. latest update with latest panda is fine. not sure why the older versions are borked, but that's irrelevant.

**Comment by dguest** [comment]
older versions are borked because we didn't keep compatibility with those when we merged #14. If anyone has a good use case for old panda we could add that back I guess.

**Comment by mswiatlo** [comment]
I hadn't updated though, so I didn't have #14 when I was doing the above. So it still should have been fine? But no big deal
